CMakeLists.txt

```diff
@@ -104,6 +104,8 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 ### Options ###
 ###############
 
+set(HUNTER_PROJECT_NAME ACF)
```

Comment:
I don't think that it's a good name for the variable. Probably it's better to rename it to something like `DRISHTI_UPLOAD_*`, `DRISHTI_COMMON_*`, `ELUCIDEYE_COMMON_*`, etc. For the latter, it's assumed that https://github.com/elucideye/drishti-upload renamed to https://github.com/elucideye/elucideye-common .

Reply:
Good point.  I promoted this to an issue here https://github.com/elucideye/acf/issues/112.